### PR TITLE
Update Razor to 6.0.0-alpha.1.20454.1

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
CC @NTaylorMullen @ajaybhargavb @TanayParikh

### CODE REVIEW

**Review by TanayParikh — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,6 +31,9 @@
 * Clean up Blazor WebAssembly notifications (PR: [#4018](https://github.com/OmniSharp/omnisharp-vscode/pull/4018))
 * Remove proposed api attribute (PR: [#4029](https://github.com/OmniSharp/omnisharp-vscode/pull/4029))
 * New completion service including override and unimported type completion (`omnisharp.enableImportCompletion`) (PR: [#3986](https://github.com/OmniSharp/omnisharp-vscode/pull/3986))
+* Updated Razor support
+  * Updated OmniSharp version (should improve stability) [dotnet/aspnetcore-tooling#20320](https://github.com/dotnet/aspnetcore/issues/20320)
+  * Corrected positioning for `@using` added by light bulb. [dotnet/aspnetcore-tooling#25019](https://github.com/dotnet/aspnetcore/issues/25019)
```
Comment (NTaylorMullen) [inline CHANGELOG.md:36]:
```suggestion
  * Corrected positioning for `@using` for components added by light bulb. [dotnet/aspnetcore-tooling#25019](https://github.com/dotnet/aspnetcore/issues/25019)
```

**Review by NTaylorMullen — APPROVED** [review]

**Comment by JoeRobich** [comment]
Yay! I fixed TravisCI. Next step is to move off of it.

**Review by JoeRobich — APPROVED** [review]

**Comment by ryanbrandenburg** [comment]
I'm holding off on merging this until I get confirmation from @TanayParikh that an issue he saw isn't reproducing after he cleans up his environment.

**Review by ajaybhargavb — APPROVED** [review]

**Review by NTaylorMullen — APPROVED** [review]

**Review by TanayParikh — APPROVED** [review]

**Comment by ryanbrandenburg** [comment]
:up::date: with the fix from our previous validation.

**Comment by NTaylorMullen** [comment]
Is it possible to produce an O# build with these bits so we can validate their end-to-end?

**Comment by JoeRobich** [comment]
> Is it possible to produce an O# build with these bits so we can validate their end-to-end?

Yup. I'll let you know when it is ready.

**Comment by JoeRobich** [comment]
@NTaylorMullen https://github.com/OmniSharp/omnisharp-vscode/releases/tag/v1.23.3-beta1

**Comment by NTaylorMullen** [comment]
You're awesome, thank you!

**Pull request merged**